operator/src/main/resources/scripts/commonMount.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Init container script for the common mount feature. 
+# See 'domain.spec.serverPod.commonMount' for details.
+
+scriptDir="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
+
+if [ "${debug}" == "true" ]; then set -x; fi;
+
+source ${scriptDir}/utils.sh
+[ $? -ne 0 ] && echo "[SEVERE] Missing file ${scriptDir}/utils.sh" && exit 1
+
+if [ -z ${COMMON_MOUNT_COMMAND} ]; then
+  trace ERROR "The common mount command $COMMON_MOUNT_COMMAND is empty. Exiting"
+  exit 1
+fi
+
+trace INFO "Common Mount: About to execute command $COMMON_MOUNT_COMMAND in container image=$CONTAINER_IMAGE. COMMON_MOUNT_PATH is $COMMON_MOUNT_PATH and COMMON_TARGET_PATH is ${COMMON_TARGET_PATH}."
```

Comment:
Minor: 1) For this and all other traces, it's more clear IMO to put single quotes around each env var.
2) You could prefix all trace text with "Common Mount:"

Reply:
Done